integrationtest/backend/lucene/src/test/java/org/hibernate/search/integrationtest/backend/lucene/LuceneExtensionIT.java

```diff
@@ -285,6 +285,19 @@ public void predicate_nativeField_nativeQuery() {
 				.hasDocRefHitsAnyOrder( INDEX_NAME, FIRST_ID );
 	}
 
+	@Test
+	public void predicate_nativeField_exists() {
+		StubMappingSearchScope scope = indexManager.createSearchScope();
+
+		IndexSearchQuery<DocumentReference> query = scope.query()
+				.asReference()
+				.predicate( f -> f.exists().onField( "nativeField" ) )
```

Comment:
`onField` does not feel natural for `exists`? But maybe consistency is better?

Reply:
Yes I'd keep it for consistency for now. If we end up implementing [HSEARCH-3522](https://hibernate.atlassian.net/browse/HSEARCH-3522), we might change the syntax to `.field( "name" ).exists()`, which in my opinion would be much better, but there should be some debate before we go there.